calculator.py

```diff
@@ -0,0 +1,51 @@
+print("Calculator")
+result = 0
+run_again = True
+
+def calculate():
+    # global keyword allows us to modify the variable outside of the current scope
+    global result
+    if operation == "+":
+        result = num1 + num2
+    elif operation == "-":
+        result = num1 - num2
+    elif operation == "/":
+        if num2 == 0:
+            print("Not a number. Try again!")
```

Comment:
**suggestion**: May be a better error message. Here 0 is a number. So the user may be confused. A better error message such "Division by zero is undefined. Enter a non-zero number." would be better